app/views/admin/resources/index.html.haml

```diff
@@ -16,14 +16,16 @@
         %tbody
           - @conference.resources.each do |resource|
             %tr
-              %td
+              %td.col_vertical_align
                 = link_to(admin_conference_resource_path(@conference.short_title, resource.id)) do
                   = resource.name
                 %td
-                  = quantity_left_of(resource)
-                %td
-                  = resource.used
-                %td
+                  %span{ id: "update_actions_#{resource.id}" }
+                    = render partial: 'update_actions', locals: { conference: @conference, resource: resource }
+                %td.vertical_align
```

Comment:
Align this column right, it will look better and it will be easier to read the numbers